modules/sdk-core/src/bitgo/trading/iTradingAccount.ts

```diff
@@ -16,7 +16,7 @@ export interface BuildPayloadAmounts {
 }
 
 export interface SignPayloadParameters {
-  payload: Payload;
+  payload: unknown;
```

Comment:
Can we type this? We get regular complaints on `any` and `unknown` from customers.

Reply:
I can make it an `object` - it can be anything compatible with `JSON.stringify`

Reply:
(that takes an `any`)

Reply:
the point of the function is to take any payload, so typing it to something more constrained would be counter-intuitive

Reply:
I changed it to `string | Record<string, unknown>` - `object` is a banned type.